documentation/ag-grid-docs/src/content/homepage/homepageFAQs.json

```
@@ -0,0 +1,42 @@
+[
+    {
+        "question": "What is the difference between AG Grid Community and Enterprise?",
+        "answer": "[AG Grid Community](./getting-started/) is the free version with basic features, while [AG Grid Enterprise](/license-pricing/) offers additional advanced features and support."
+    },
+    {
+        "question": "Can I create a React Table for free?",
+        "answer": "Yes, you can create a React Table for free using our open-source library, [AG Grid Community](./getting-started/)."
+    },
+    {
+        "question": "Will my React Tables be compatible with AG Charts?",
+        "answer": "Yes, AG Grid is designed to be compatible with AG Charts, allowing seamless integration between React Tables and [React Charts](/charts/react/quick-start/)."
```

This link isn't going to work on staging. To make it work, we might need to add a charts website identifier eg,

```suggestion
        "answer": "Yes, AG Grid is designed to be compatible with AG Charts, allowing seamless integration between React Tables and [React Charts](%charts%/react/quick-start/)."
```

Maybe for another PR though

---

For now making the charts link point to prod solves issue